src/core/config/key_values.rs

```diff
@@ -3,27 +3,40 @@ use std::ops::Deref;
 
 use serde::{Deserialize, Deserializer, Serialize, Serializer};
 
+use crate::core::is_default;
+
+type Value = (String, Option<bool>);
+
 #[derive(Clone, Debug, Default, Eq, PartialEq, schemars::JsonSchema)]
-pub struct KeyValues(pub BTreeMap<String, String>);
+pub struct KeyValues(pub BTreeMap<String, Value>);
```

Comment:
```suggestion
pub struct KeyValues<Value>(BTreeMap<String, Value>);
```
Make it polymorphic.

Reply:
Value is not something from JL, it's `type Value = (String, Option<bool>);`